runtime/src/lib.rs

```diff
@@ -767,6 +767,39 @@ impl substrate_pallet_multi_account::Trait for Runtime {
     type MaxSignatories = MultiAccountMaxSignatories;
 }
 
+parameter_types! {
+    pub const TreasuryModuleId: ModuleId = ModuleId(*b"cen/trsy");
```

Comment:
What are `kusama` settings for all these? I would take them as a reference.

Reply:
I reused the default values from kusama. Should we discuss the values with Cassidy? 
I think for example the TipFindersFee shouldn't be 20%.

Reply:
I created a hackMD to discuss all config parameter